components/StyledDatePickerController/StyledDatePickerController.jsx

```diff
@@ -37,8 +37,8 @@ export const StyledDatePickerController =
           focused={true}
           isDayHighlighted={isDayHighlighted}
           isDayBlocked={isDayBlocked}
-          navPrev={disablePrev ? <span /> : <Icon className="universe-chevron-left" />}
-          navNext={disableNext ? <span /> : <Icon className="universe-chevron-right" />}
+          navPrev={disablePrev ? <span /> : null}
+          navNext={disableNext ? <span /> : null}
```

Comment:
It looks like this removes the previous / next icon buttons? It will now always be either an empty span or null?

Reply:
By passing null it takes on default behaviour and that's what we want.